composer.json

```diff
@@ -12,8 +12,9 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
-        "wp-cli/wp-cli": "^2"
+        "php": ">=7.1",
+        "wp-cli/wp-cli": "^2",
+        "inmarelibero/gitignore-checker": "1.0.2"
```

Comment:
The difference between v1.0.1 and v1.0.2 was a feature not a security release so the maintainer isn't following Semver exactly. It's still _probably_ safe, particularly since we have tests covering what issues in this repo have requested/reported.